src/data/ingredients-sample.js

```diff
@@ -0,0 +1,102 @@
+export const ingredientsData = [
```

Comment:
It looks like these files are in your other PR, too. Depending on which one gets merged first, just make sure you update the one that hasn't been merged.